pilot/test/integration/cloudfoundry/copilot_test.go

```diff
@@ -52,7 +52,39 @@ const (
 	appEnvoyListener  = "http://127.0.0.1:6666"
 )
 
-func TestEdgeRouterWithMockCopilot(t *testing.T) {
+var routeRule = `
```

Comment:
shouldn't these be in yaml files in testdata

Reply:
I like the idea of having them in the test way better (I can actually see the data). Fine either way but I think it reads better when someone can look at the details of the config inline.